app/uk/gov/hmrc/cataloguefrontend/service/CostEstimationService.scala

```diff
@@ -58,21 +61,65 @@ object CostEstimationService {
       Json.reads[DeploymentConfig]
   }
 
-  final case class CostEstimation(totalSlots: Int) {
+  final case class ServiceCostEstimate(forEnvironments: List[ServiceCostEstimate.ForEnvironment]) {
 
-    lazy val yearlyCostGbp: Double =
-      totalSlots * 650
+    lazy val summary: ServiceCostEstimate.Summary =
+      forEnvironments
+        .foldLeft(Summary.zero)((summary, forEnv) => summary + Summary(forEnv.slots, forEnv.yearlyCostGbp))
   }
 
-  object CostEstimation {
+  object ServiceCostEstimate {
 
-    def fromDeploymentConfigByEnvironment(deploymentConfigByEnvironment: DeploymentConfigByEnvironment): CostEstimation = {
-      val totalSlots =
-        deploymentConfigByEnvironment.values
-          .map(c => c.slots * c.instances)
-          .sum
+    def fromDeploymentConfigByEnvironment(
+      deploymentConfigByEnvironment: DeploymentConfigByEnvironment,
+      serviceCostEstimateConfig: ServiceCostEstimate.Config): ServiceCostEstimate =
+      ServiceCostEstimate {
+        deploymentConfigByEnvironment
+          .collect { case (env, config) if config.slots > 0 && config.instances > 0 =>
+            val yearlyCostGbp =
+              config.slots * config.instances * serviceCostEstimateConfig.slotCostPerYear
+            ForEnvironment(env, config.slots, yearlyCostGbp)
+          }
+          .toList
+          .sortBy(_.environment)
+      }
 
-      CostEstimation(totalSlots)
+    final case class ForEnvironment(
+      environment: Environment,
+      slots: Int,
+      yearlyCostGbp: Double
+    )
+
+    final case class Summary(totalSlots: Int, totalYearlyCostGbp: Double) {
+      def +(other: Summary): Summary =
+        Summary(totalSlots + other.totalSlots, totalYearlyCostGbp + other.totalYearlyCostGbp)
+    }
+
+    object Summary {
+
+      val zero: Summary =
+        Summary(totalSlots = 0, totalYearlyCostGbp = 0)
+    }
+
+    final case class Config(slotCostPerYear: Double, totalAwsCostPerYear: String)
+
+    object Config {
+
+      val defaultSlotCostPerYear: Double = 650
+      val defaultTotalAwsCostPerYear: String = "£5.4M"
+
+      def get(): Config = {
+        val slotCostPerYear =
+          sys.props.get("cost-estimates.slot-cost-per-year")
```

Comment:
can we remove this `get` and inject the Config where needed? Then it can use configuration in config files as well as in system properties - and make testing easier etc..